src/software/thunderscope/robot_diagnostics/chicker.py

```diff
@@ -140,6 +140,8 @@ def enable_kick_chip_buttons(self):
         """
         if self.no_auto_selected:
             self.kick_chip_buttons_enable = True
+            power_control = PowerControl()
+            self.proto_unix_io.send_proto(PowerControl, power_control, True)
```

Comment:
this can be a bit confusing for someone not familiar with how the buffer implementation works. Could you move this call and the second empty proto call to a function and add a documentation there?